doc/telemetry/README.md

```diff
@@ -0,0 +1,81 @@
+Telemetry Documentation
+=======================
+
+> Two types of telemetry are used to monitor and analyze execution of Azure CLI commands. One called ARM telemetry is recorded basing on HTTP traffic by ARM, and another is client side telemetry sent by Azure CLI.
+
+### ARM Telemetry
+ARM telemetry tracks all HTTP requests and responses through ARM endpoint. As far as we know, below cases don't have ARM telemetry record.
+- Command doesn't create request successfully, for instance, parameter cannot pass validation, request or payload cannot be constructed.
+- Command calls data plane service API.
+- Network is inaccessible.
+- No request is needed during execution.
+
+Kusto Cluster and Database: https://dataexplorer.azure.com/clusters/armprod/databases/ARMProd
+
+
+### CLI Client Telemetry
+Client side telemetry is sent at the end of Azure CLI command execution. It covers all commands, no matter if it has http requests or just has local operations.
+Sanitized data is stored in Kusto cluster which is managed by DevDiv Data team.
+
+Kusto Cluster and Database: https://dataexplorer.azure.com/clusters/ddazureclients/databases/AzureCli
+
+All Azure CLI data is stored in a large json named `Properties` in table `RawEventsAzCli`. Some properties are flatten, some are not. Here are some useful fields:
+> The telemetry has different schema pre Azure CLI 2.0.28. All the fields explained below are for new schema, in other words, CLI version > 2.0.28.
+- `EventName`: `azurecli/command` or `azurecli/fault` or `azurecli/extension`
+    - `azurecli/command` means this is common event record with general `Properties` field.
+    - `azurecli/fault` means this is additional event record with extra exception info in `Properties` field.
+    - `azurecli/extension` means this is additional event record with customized info in `Properties` field.
+    - Additional event record can be joined with common event record using `CorrelationId`.
+- `CorrelationId`: GUID to join additional event records with common event record
```

Comment:
Would be good to include description/definition of EventId as well.

Reply:
EventId is just GUID